packages/auth/src/accessToken/service.ts

```diff
@@ -72,16 +89,41 @@ async function introspect(
   }
 }
 
-async function revoke(
-  deps: ServiceDependencies,
-  id: string
-): Promise<Error | undefined> {
+async function revoke(deps: ServiceDependencies, id: string): Promise<void> {
   const token = await AccessToken.query(deps.knex).findById(id)
   if (token) {
-    if (!isTokenExpired(token)) {
-      await token.$query(deps.knex).patch({ expiresIn: 1 })
+    await token.$query(deps.knex).delete()
+  }
+}
+
+async function rotate(
+  deps: ServiceDependencies,
+  managementId: string
+): Promise<Rotation> {
+  let token = await AccessToken.query(deps.knex).findOne({ managementId })
+  if (token) {
+    await token.$query(deps.knex).delete()
+    const newManagementId = uuid()
+    token = await AccessToken.query(deps.knex).insertAndFetch({
+      value: uuid(),
```

Comment:
I'm currently planning to generate access token values [like thus](https://github.com/interledger/rafiki/blob/8cdbc80efc4e823fa7137a6514d50edea39b8d6a/packages/auth/src/accessToken/service.ts#L109), it differentiates them from identifiers that should be used in URLs and is a bit closer to the preconception of a token (though that may just be me).